docker-compose-integration.yaml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+
+services:
+  test:
+    image: elixir:latest
+    command: bash -c "cd app/ && mix local.hex --force && mix local.rebar --force && mix deps.get && mix test --warnings-as-errors"
+    environment:
+      - MIX_ENV=integration_test
+    volumes:
+      - .:/app
+    links:
+      - jellyfish
+  jellyfish:
+    image: "ghcr.io/jellyfish-dev/jellyfish:${TAG:-edge}"
+    container_name: jellyfish
+    restart: on-failure
+    healthcheck:
+      test: >
+        curl --fail -H "authorization: Bearer development" http://localhost:5002/room || exit 1
+      interval: 3s
+      retries: 30
+      timeout: 2s
+    environment:
+      VIRTUAL_HOST: "localhost"
+      USE_INTEGRATED_TURN: "true"
+      INTEGRATED_TURN_IP: "${INTEGRATED_TURN_IP:-127.0.0.1}"
+      INTEGRATED_TURN_PORT_RANGE: "50000-65355"
+      INTEGRATED_TCP_TURN_PORT: "49999"
+      SERVER_API_TOKEN: "development"
+      PORT: 5002
+      SECRET_KEY_BASE: "super-secret-key"
+    ports:
+      - "5002:5002"
```

Comment:
Are you sure you don't need to explicitly expose UDP ports or use `network: host` here?